backend/entities/Partials/BaseContext.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.EntityFrameworkCore;
+namespace Pims.Dal
+{
+    /// <summary>
+    /// Partial PimsBaseContext class that provides a protected constructor that allows for inheritance with a dynamic DbContextOptions object
+    /// </summary>
+    public partial class PimsBaseContext : DbContext
```

Comment:
Allows inheritance of the context generated by the scaffold. Scaffold now generates PimsBaseContext instead of PimsContext